### PR TITLE
Gusinacio/unique and timestamp check

### PR DESCRIPTION
**Pull request by gusinacio**

### CODE REVIEW

File: tap_core/src/tap_receipt/checks.rs [hunk 1]
```diff
@@ -70,6 +78,58 @@ impl Check for TimestampCheck {
...
+        receipts: Vec<ReceiptWithState<Checking>>,
+    ) -> (
+        Vec<ReceiptWithState<Checking>>,
+        Vec<ReceiptWithState<Failed>>,
+    ) {
+        let (mut checking, mut failed) = (vec![], vec![]);
+        for receipt in receipts.into_iter() {
+            let receipt_timestamp_ns = receipt.signed_receipt().message.timestamp_ns;
+            let min_timestamp_ns = self.0;
+            if receipt_timestamp_ns >= min_timestamp_ns {
```
Comment (aasseman) [inline tap_core/src/tap_receipt/checks.rs:95]:
```suggestion
            if receipt_timestamp_ns > min_timestamp_ns {
```

**Review by aasseman — CHANGES_REQUESTED** [review]

File: tap_core/src/tap_receipt/checks.rs (hunk 1 above)
Comment (gusinacio) [reply tap_core/src/tap_receipt/checks.rs:95]: done

Comment (aasseman) [reply tap_core/src/tap_receipt/checks.rs:95]: Actually was correct but needs a docstring explaining what timestamp check it does

Comment (gusinacio) [reply tap_core/src/tap_receipt/checks.rs:96]: just added the docstring

**Review by aasseman — APPROVED** [review]

**Review by aasseman — APPROVED** [review]

**Pull request merged**